test/integration/policy_pod_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+const (
+	policyPodName   = "policy-pod"
+	policyPodURL    = policyCollectCMURL + policyPodName + ".yaml"
+	policyPodNSName = "policy-pod"
+	podName         = "sample-nginx-pod"
+	policyPodKind   = "Pod"
+)
+
+var podLabel = Label("policy-collection", "stable", "SVT")
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the "+policyPodName+" policy", podLabel, func() {
+	It("stable/"+policyPodName+" should be created on the Hub", func() {
+		By("Creating the policy on the Hub")
+		_, err := utils.KubectlWithOutput(
+			"apply", "-f", policyPodURL, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		By("Creating the " + policyPodNSName + " namespace on the managed cluster")
+		namespace := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: policyPodNSName}}
```

Comment:
```suggestion
namespace := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: policyPodNSName, Labels: map[string]string{"e2e": "true"}}}
```
This would make it easier to clean up the namespace in [clean up script](https://github.com/stolostron/governance-policy-framework/blob/b29a7e4bf832416d290ccdf93b401a744fa86988/build/clean-up-cluster.sh#L62) if test failed in the half way.